src/Admin.php

```diff
@@ -262,4 +265,13 @@ public static function admin_head() {
 EOD;
   }
 
+  /**
+   * @implements wp_dropdown_users_args
+   */
+  public static function wp_dropdown_users_args($query_args) {
+    $query_args['who'] = 'author';
```

Comment:
Instead of setting the parameter, you need to use it as condition.  Do you understand why?